tests/test_apiutils.py

```diff
@@ -230,6 +203,18 @@ def test_get_invalid_named_group_key(capsys):
             assert 'was not found' in out
 
 
+@patch('ConfigParser.ConfigParser')
+def test_replace_loggroup_section(mocked_configparser_class):
```

Comment:
@ilyabiryukov-r7 
Hi - this unit test was giving problems but I then got it passing - just not 100% if its solid - a review would be great - Thanks